tests/Fixer/PhpUnit/PhpUnitStaticAssertionCallsFixerTest.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\PhpUnit;
+
+use PhpCsFixer\Fixer\PhpUnit\PhpUnitStaticAssertionCallsFixer;
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+use PHPUnit\Framework\Assert;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\PhpUnit\PhpUnitStaticAssertionCallsFixer
+ */
+final class PhpUnitStaticAssertionCallsFixerTest extends AbstractFixerTestCase
+{
+    public function testFixerContainsAllPhpunitStaticMethodsInItsList()
+    {
+        $assertionRefClass = new \ReflectionClass(Assert::class);
+        $updatedStaticMethodsList = $assertionRefClass->getMethods(\ReflectionMethod::IS_STATIC | \ReflectionMethod::IS_PUBLIC);
+
+        $fixerRefClass = new \ReflectionClass(PhpUnitStaticAssertionCallsFixer::class);
+        $staticMethodsProperty = $fixerRefClass->getProperty('staticMethods');
+        $staticMethodsProperty->setAccessible(true);
+        $staticMethods = $staticMethodsProperty->getValue();
+        $staticMethodsProperty->setAccessible(false);
+
+        foreach ($updatedStaticMethodsList as $method) {
```

Comment:
What about using data provider instead of `foreach`?

Reply:
What would be the added value of having 155 tests on 1 data, instead of 1 test with 155 data?
Unlike the tests that use the `doFix` helper, here there is no environoment to set up, nor diff to output in case of fail.

Reply:
It would separate preparing test data from actually testing it, plus if more than 1 check fail all problems would be displayed at once.

Reply:
Now all the eventual missing methods are all listed at once.